packages/engine-server/src/markdown/remark/dendronPub.ts

```diff
@@ -152,13 +152,16 @@ function plugin(this: Unified.Processor, opts?: PluginOpts): Transformer {
         }
 
         const copts = opts?.wikiLinkOpts;
-        if (note && opts?.transformNoPublish) {
+        if (!note && opts?.transformNoPublish) {
+          value = "403";
```

Comment:
minor nit: use http status codes constants for this.

Reply:
created a separate PR for this here -> https://github.com/dendronhq/dendron/pull/1344